ddtrace/contrib/kafka/patch.py

```diff
@@ -66,6 +68,23 @@ def __bool__(self):
     __nonzero__ = __bool__
 
 
+class TracedSerializingProducer(_SerializingProducer):
+    def __init__(self, config, *args, **kwargs):
+        super(TracedSerializingProducer, self).__init__(config, *args, **kwargs)
+        self._dd_bootstrap_servers = (
+            config.get("bootstrap.servers")
+            if config.get("bootstrap.servers") is not None
+            else config.get("metadata.broker.list")
+        )
+
+    # in older versions of confluent_kafka, bool(Producer()) evaluates to False,
+    # which makes the Pin functionality ignore it.
+    def __bool__(self):
+        return True
+
+    __nonzero__ = __bool__
```

Comment:
I don't know if this is necessary - if SerializingProducer was around in these "older versions"